reps_event_app/lib/ui/eventsDetails.dart

```diff
@@ -0,0 +1,33 @@
+import "package:flutter/material.dart";
+import 'package:reps_event_app/models/events_model.dart';
+
+class EventsDetails extends StatelessWidget {
+  final EventsModel events;
+
+  EventsDetails({this.events});
+ 
+
+  bodyWidget(BuildContext context) => Container(
+        child: Column(
```

Comment:
Use `ListView` instead of `Column`. The bottom will be overflowed for devices with a smaller screen size

Reply:
okay, this is just basic, will change it in the final design :smile: